torch/onnx/utils.py

```diff
@@ -288,7 +288,7 @@ def _trace(func, args, operator_export_type, return_outs=False):
         args = (args, )
 
     trace_graph, torch_out, inputs_states = \
-        torch.jit._get_trace_graph(func, args, _force_outplace=False, _return_inputs_states=True)
+        torch.jit._get_trace_graph(func, args, strict=False, _force_outplace=False, _return_inputs_states=True)
```

Comment:
@houseroad I am relaxing the constraints of onnx exporter to use this `strict=False` flag, let me know if you see any issues from this.